paddle/fluid/framework/ir/graph_pattern_detector.h

```diff
@@ -419,6 +419,23 @@ struct FC : public PatternBase {
   PATTERN_DECL_NODE(Out);
 };
 
+// Embedding
+struct Embedding : public PatternBase {
+  Embedding(PDPattern* pattern, const std::string& name_scope)
+      : PatternBase(pattern, name_scope, "embedding") {}
+
+  PDNode* operator()(PDNode* x);
+
+  // declare operator node's name
+  PATTERN_DECL_NODE(lookup_table);
+  // Inputs
+  //
```

Comment:
remove this line.

Reply:
When : 
*PATTERN_DESC_NODE(lookup_table)*
is removed then :
*GET_IR_NODE_FROM_SUBGRAPH(lookup_table, lookup_table, embedding_pattern);*
does not compile  and without that line I cannot get lookup table attributes eg. is_sparse , is_distributed . Currently I need to know those attributes  so I can not to execute pattern replacement when either of them is set.